bodhi/server/config.py

```diff
@@ -443,6 +443,9 @@ class BodhiConfig(dict):
         'waiverdb.access_token': {
             'value': None,
             'validator': _validate_none_or(six.text_type)},
+        'koji_web_url': {
+            'value': 'https://koji.fedoraproject.org',
+            'validator': str},
```

Comment:
```urljoin()``` will behave differently if this setting ends in a ```/``` than if it does not:

```
>>> u('https://example.com/one', 'thing/other_thing')
'https://example.com/thing/other_thing'
>>> u('https://example.com/one/', 'thing/other_thing')
'https://example.com/one/thing/other_thing'
```

To avoid exposing Bodhi admins to this, let's add a new validator in this module that ensures that the setting ends in a ```/```. This way, Bodhi will allow the admin to set ```https://koji.fedoraproject.org/koji``` here and it should still work.